contactaccessrights/js/src/access-rights/services/location.js

```diff
@@ -0,0 +1,19 @@
+define([
```

Comment:
Put the apis in `access-rights/services/api` (or maybe `accesss-rights/apis`? not sure :/)